packages/forms-web-app/src/dynamic-forms/middleware/get-journey-response-for-appellant.js

```diff
@@ -32,7 +34,9 @@ module.exports = async (request, response, next) => {
 		logger.error(error);
 	}
 
-	if (!submission) return response.status(404).render('error/not-found');
+	if (!submission || !(await isFeatureActive(FLAG.APPEAL_FORM_V2, submission.LPACode))) {
```

Comment:
replacing the middleware check with a check here so we don't need to keep the old req.session.appeal data